Base/usr/share/man/man8/userdel.md

```diff
@@ -0,0 +1,47 @@
+## Name
+
+userdel - delete a user account
+
+## Synopsis
+
+```**sh
+# userdel [-r] <login>
+```
+
+## Description
+
+This program delete a user account in the system.
+
+This program must be run as root.
+
+## Options
+
+* `-r`, `--remove`: Remove the home directory for this user if the directory exists.
+
+## Exit Values
+
+* 0 - Success
+* 1 - Couldn't update the password file
+* 6 - Specified user doesn't exist
+* 12 - Couldn't remove home directory
+
+## Files
+
+* `/etc/passwd` - user information (such as UID and GID) in this file is deleted.
+* `/home/` - user home directroy is deleted if the `-r` flag is specified.
+
+## Examples
+
+```sh
+# userdel alice
+# userdel -r alice
+# userdel --remove alice
+```
+
+## Author
+
+Fei Wu <f.eiwu@yahoo.com>
```

Comment:
We haven't been putting authorship info into man pages in Serenity so far, but I suppose it's fine? @awesomekling what do you say?

Reply:
I would prefer to avoid that. It's nice having copyrights in source code but they look a bit out of place in documentation and GUI.

Reply:
OK, I will delete it.